kapitan/inputs/copy.py

```diff
@@ -25,17 +26,22 @@ def compile_file(self, file_path, compile_path, ext_vars, **kwargs):
         path can be either a file or directory.
         """
 
+        # Whether to fail silently if the path does not exists.
+        ignore_missing = self.ignore_missing
         try:
             logger.debug("Copying {} to {}.".format(file_path, compile_path))
-            if os.path.isfile(file_path):
-                if os.path.isfile(compile_path):
-                    shutil.copy2(file_path, compile_path)
+            if os.path.exists(file_path):
+                if os.path.isfile(file_path):
+                    if os.path.isfile(compile_path):
+                        shutil.copy2(file_path, compile_path)
+                    else:
+                        os.makedirs(compile_path, exist_ok=True)
+                        shutil.copy2(file_path, os.path.join(compile_path, os.path.basename(file_path)))
                 else:
-                    os.makedirs(compile_path, exist_ok=True)
-                    shutil.copy2(file_path, os.path.join(compile_path, os.path.basename(file_path)))
-            else:
-                compile_path = os.path.abspath(compile_path)  # Resolve relative paths
-                copy_tree(file_path, compile_path)
+                    compile_path = os.path.abspath(compile_path)  # Resolve relative paths
+                    copy_tree(file_path, compile_path)
+            elif ignore_missing == False:
+                raise OSError(f"Path {file_path} does not exist and `ignore_missing` is {ignore_missing}")
```

Comment:
I guess it's better to raise CompileError to keep kapitan error handling/printing consistent

Reply:
@ramaro I used the same error that was already raised in line 39. Shall I also change that?